bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1179 @@
+    cmd = [str(backend_python), str(Path(__file__).resolve()), *sys.argv[1:]]
+    result = subprocess.run(cmd, cwd=str(PROJECT_ROOT), env=env)
+    raise SystemExit(result.returncode)
+
+
+def utcnow_iso() -> str:
+    return datetime.now(timezone.utc).isoformat()
+
+
+def log(message: str) -> None:
+    safe_message = message.replace("\r", " ").replace("\n", " ")
+    info(f"[{datetime.now().strftime('%H:%M:%S')}] {safe_message}")
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)